DataLakeModels/Models/Reels/ClipsMeta.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Runtime;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace DataLakeModels.Models.Reels {
+    public class ClipsMeta : IEquatable<ClipsMeta> {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+        public Reel Reel { get; set; }
+        public string ReelId { get; set; }
+        public string NuxInfo { get; set; }
```

Comment:
Do you know what this encodes? I think we are hardly going to understand what all of the fields mean, but maybe we could start a default way of documenting the ones that we understand